packages/cli.cmd.typescript/src/generate/handleGenerate.mts

```diff
@@ -256,17 +254,15 @@ async function generateSourceFiles(
   ontology: WireOntologyDefinition,
   fs: MinimalFs,
 ) {
-  await (args.beta
-    ? generateClientSdkVersionTwoPointZero
-    : generateClientSdkVersionOneDotOne)(
-      ontology,
-      getUserAgent(args.version),
-      fs,
-      args.outDir,
-      args.packageType,
-      args.ontologyApiNamespace,
-      args.apiNamespaceMap,
-    );
+  await generateClientSdkVersionTwoPointZero(
```

Comment:
Not related to this, but looking at `handleGenerate` in this file, should we still have a beta flag for it?

Reply:
good call , we can clean that up now too